client/src/stores/counter.js

```diff
@@ -0,0 +1,12 @@
+import { ref, computed } from "vue";
+import { defineStore } from "pinia";
+
+export const useCounterStore = defineStore("counter", () => {
```

Comment:
Will the convention be to use double quotes instead of single quotes? Or both?

Reply:
Convention will be double quotes. I prefer them over single quotes. Will configure that in prettier as a part of next PR.